frontend/src/pages/Properties/PropertyDetails.tsx

```diff
@@ -103,19 +105,23 @@ const PropertyDetails = (props: any) => {
         <div>Loading.....</div>
       ) : (
         <PropertyContainer>
-          <ThumbNail
-            src='https://www.samplemcdougald.org/wp-content/uploads/2017/10/visit-sample-mcdougald-300x300.jpg'
-            alt='house'
-          />
-          <Top>
-            <MainText data-testid='house-detail'>{property.name}</MainText>
-            <SecondaryText>{property.address}</SecondaryText>
-            <BackButton text='Edit Property' colour='var(--colour-accent)' />
-            <BackButton
-              text='Go Back'
-              colour='var(--colour-accent)'
-              onClick={() => props.history.push('/properties')}
-            />
+					<Top>
```

Comment:
Same issue as above, can you fix this please?

Reply:
I think it looks better now!